app/src/main/java/waiter/Socket/SocketWrapper.java

```diff
@@ -1,11 +1,10 @@
-package waiter.Interactive;
+package waiter.Socket;
 
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
-import java.net.Socket;
 
-public record Interactor(Socket socket) implements Interactive {
+public record SocketWrapper(java.net.Socket socket) implements Socket {
```

Comment:
Non-blocking: If you're trying to remove all instances of `*Wrapper` perhaps this could be changed to `ClientSocket`.